src/App.jsx

```
@@ -8,99 +1,445 @@
+                isOpen={isThemeSelectorOpen}
+                onClose={() => setIsThemeSelectorOpen(false)}
+              />
+            )}
+          </AnimatePresence>
+        </motion.div>
+
+        {/* Enhanced Performance Monitoring */}
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+              // Enhanced Core Web Vitals monitoring
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)